src/deploy/carvel/register-apps.sh

```diff
@@ -43,10 +43,12 @@ else
     # unfortunately different in docker image names and registration link.
     BROKER_NAME=rabbitmq
 fi
-
 if [ "$STREAM_APPS_RT_VERSION" = "" ]; then
-    STREAM_URI="https://dataflow.spring.io/$BROKER_NAME-${TYPE}-latest"
-elif [[ "$STREAM_APPS_RT_VERSION" = *"SNAPSHOT"* ]]; then
+    # export STREAM_APPS_RT_VERSION=2021.1.2 # release for Boot 2.x
+    export STREAM_APPS_RT_VERSION=2022.0.0
```

Comment:
These are still SB3.x based apps. `2022.0.0 == 4.0 stream apps`

Reply:
The docs specify that the default registered by the scripts are 2022.0.0

Reply:
Where in the docs, we need to update that because I don't think that is accurate.